applications/openshift/api-server/api_server_api_priority_gate_enabled/rule.yml

```diff
@@ -33,9 +33,6 @@ identifiers:
 severity: medium
 
 references:
-  cis@ocp4: 1.2.9
-  nerc-cip: CIP-003-8 R6,CIP-004-6 R3,CIP-007-3 R6.1
-  nist: CM-6,CM-6(1)
   pcidss: Req-2.2
   srg: SRG-APP-000516-CTR-001325
```

Comment:
How come we still include this rule for STIG?

Reply:
Er, this is the SRG with a bunch of extra rules.
They will be removed soon.